lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -2436,9 +2436,12 @@ size_t
 TypeSystemSwiftTypeRef::GetNumTemplateArguments(opaque_compiler_type_t type) {
   return m_swift_ast_context->GetNumTemplateArguments(ReconstructType(type));
 }
+
 CompilerType
 TypeSystemSwiftTypeRef::GetTypeForFormatters(opaque_compiler_type_t type) {
-  return m_swift_ast_context->GetTypeForFormatters(ReconstructType(type));
+  auto impl = [&]() -> CompilerType { return {this, type}; };
```

Comment:
If this isn't a placeholder, we could move it into TypeSystemSwift.

Reply:
I haven't lost track of this, I will move it as soon as it's non-breaking to do so.